### DISCUSSION TITLE
Example for platform library projects

### DISCUSSION THREAD

**Discussion by jslew** [mattleibow/DeviceRunners#18] [Q&A]
Hello @mattleibow, very helpful stuff here. I have a (newly-converted-from Xamarin)  net7.0-ios, net7.0-android class libraries, with their own (NUnit) unit test projects that I would like to be able to run on-device.  Any guidance you can give for how to accomplish this setup?
Thanks!

**Comment by mattleibow** [comment]
I have a sample app that has both Xunit and Nunit test.

I would expect a simple test app with the tests that has a project reference to the library?
